src/prototype/components/AlertContainer.tsx

```
@@ -0,0 +1,62 @@
+  // Extract the alert title from markdown content
+  const getAlertTitle = (content: string): string => {
+    // Look for patterns like "⚠️ **Goal Drift Alert**" or "📊 **Sprint Planning Check**"
+    const titleMatch = content.match(/^.*?\*\*(.*?)\*\*/);
+    if (titleMatch) {
+      // Remove emoji and clean up
+      return titleMatch[1].trim();
+    }
+    
+    // Fallback: look for first line
+    const firstLine = content.split('\n')[0].trim();
+    return firstLine.replace(/[⚠️📊🔄]/g, '').replace(/\*\*/g, '').trim();
```

Biome (2.1.2)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)